httpserver/service_node.h

```diff
@@ -230,10 +238,22 @@ class ServiceNode {
         std::vector<service_node::storage::Item>& all_entries) const;
 
     // Return the current PoW difficulty
-    int get_pow_difficulty() const;
+    pow_difficulty_t get_pow_difficulty() const;
 
     bool retrieve(const std::string& pubKey, const std::string& last_hash,
                   std::vector<service_node::storage::Item>& items);
+
+    void
+    set_difficulty_history(const std::vector<pow_difficulty_t>& new_history) {
+        curr_pow_difficulty_ =
```

Comment:
This is not needed since you already have a default value for `curr_pow_difficulty_`?